model/src/main/kotlin/licenses/License.kt

```diff
@@ -21,12 +21,14 @@ package org.ossreviewtoolkit.model.licenses
 
 import com.fasterxml.jackson.annotation.JsonInclude
 
-import java.util.SortedSet
-
 import org.ossreviewtoolkit.spdx.SpdxSingleLicenseExpression
 
 /**
  * A class for configuring meta data for a specific license referred to by a SPDX license identifier.
+ *
+ * The meta data consists of assignments to generic categories whose exact meaning is customer specific.
+ * The categories a license belong to can be evaluated by other components, such as rules or templates,
+ * which can decide - based on this information - how to handle a specific license.
  */
 data class License(
```

Comment:
Another note to myself: I believe we should also consider renaming this class e.g. to `LicenseCategorization` in a follow-up PR.